spec/controllers/cloud_subnet_controller_spec.rb

```diff
@@ -1,88 +1,86 @@
-      @tag2 = FactoryGirl.create(:classification_tag,
-                                 :name   => "tag2",
-                                 :parent => classification)
-      allow(Classification).to receive(:find_assigned_entries).with(@ct).and_return([@tag1, @tag2])
+    before do
+      stub_user(:features => :all)
       session[:tag_db] = "CloudSubnet"
-      edit = {
-        :key        => "CloudSubnet_edit_tags__#{@ct.id}",
+      session[:edit] = {
+        :key        => "CloudSubnet_edit_tags__#{ct.id}",
         :tagging    => "CloudSubnet",
-        :object_ids => [@ct.id],
+        :object_ids => [ct.id],
         :current    => {:assignments => []},
-        :new        => {:assignments => [@tag1.id, @tag2.id]}
+        :new        => {:assignments => [tag1.id, tag2.id]}
       }
-      session[:edit] = edit
-    end
-
-    after(:each) do
-      expect(response.status).to eq(200)
+      FactoryGirl.create(:tagging, :tag => tag1.tag, :taggable => ct)
+      FactoryGirl.create(:tagging, :tag => tag2.tag, :taggable => ct)
     end
 
     it "builds tagging screen" do
-      post :button, :params => { :pressed => "cloud_subnet_tag", :format => :js, :id => @ct.id }
+      post :button, :params => { :pressed => "cloud_subnet_tag", :format => :js, :id => ct.id }
+
       expect(assigns(:flash_array)).to be_nil
+      expect(response.status).to eq(200)
     end
 
     it "cancels tags edit" do
-      session[:breadcrumbs] = [{:url => "cloud_subnet/show/#{@ct.id}"}, 'placeholder']
-      post :tagging_edit, :params => { :button => "cancel", :format => :js, :id => @ct.id }
+      session[:breadcrumbs] = [{:url => "cloud_subnet/show/#{ct.id}"}, 'placeholder']
+
+      post :tagging_edit, :params => { :button => "cancel", :format => :js, :id => ct.id }
+
       expect(assigns(:flash_array).first[:message]).to include("was cancelled by the user")
       expect(assigns(:edit)).to be_nil
+      expect(response.status).to eq(200)
     end
 
     it "save tags" do
-      session[:breadcrumbs] = [{:url => "cloud_subnet/show/#{@ct.id}"}, 'placeholder']
-      post :tagging_edit, :params => { :button => "save", :format => :js, :id => @ct.id }
+      session[:breadcrumbs] = [{:url => "cloud_subnet/show/#{ct.id}"}, 'placeholder']
+
+      post :tagging_edit, :params => { :button => "save", :format => :js, :id => ct.id }
+
       expect(assigns(:flash_array).first[:message]).to include("Tag edits were successfully saved")
       expect(assigns(:edit)).to be_nil
+      expect(response.status).to eq(200)
     end
   end
 
   describe "#show" do
-    before do
-      EvmSpecHelper.create_guid_miq_server_zone
-      @subnet = FactoryGirl.create(:cloud_subnet)
-      login_as FactoryGirl.create(:user)
-    end
+    let(:subnet) { FactoryGirl.create(:cloud_subnet) }
 
-    subject do
-      get :show, :params => {:id => @subnet.id}
-    end
+    before { login_as FactoryGirl.create(:user) }
+
+    subject { get :show, :params => {:id => subnet.id} }
```

Comment:
It feels a little inconsistent to me, can't you make it more similar to the other specs where you don't have `subject` and test against the `response`.